builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -398,6 +396,31 @@ node() {
   }
 }
 
+def attachTagstoLogGroup(config, environment_logical_id, credsId){
+  try {
+    def deploymentDescriptor = null
+    if( env_deployment_descriptor != null ){
+      deploymentDescriptor = env_deployment_descriptor
+    } else {
+      deploymentDescriptor = config['deployment_descriptor']
+    }
+    // If no descriptor present then simply making an empty one. The readYaml default behavior is to return empty string back that is harmful as Map not String is expected below
+    def doc = deploymentDescriptor ? readYaml(text: deploymentDescriptor) : [:]
+    def functionsElem = doc['functions']
+
+    // changing variable names appropriately
+    def serviceDomain = "${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}"
+    functionsElem.each{functionName, funcConfig ->
+      def lambdaName = funcConfig['name'] ? funcConfig['name'] : "${serviceDomain}-${functionName}-${environment_logical_id}"
```

Comment:
Is it possible to get the function from the `serverless info..` or `cloudformation describe..` output? We're still 'hoping'  that the function name will follow this convention while we rely on serverless.yml for deploying infrastructure.